Makefile.am

```diff
@@ -1,4 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4
+AM_CPPFLAGS = $(shell pkg-config --cflags libjxl)
+
 
```

Comment:
Please remove this. All needed variables are set in `configure.ac`. Do not call `pkg-config` in `Makefile.am`.